### ISSUE TITLE
Runtime panic when using local mode on macos

### ISSUE THREAD

**Issue by local-administrator** [mitmproxy/mitmproxy#7043]
### Problem Description
Running `mitmproxy --mode local` results in a runtime panic

`❯ mitmproxy --mode local`
> thread '<unnamed>' panicked at mitmproxy-rs/src/server/local_redirector.rs:118:30:
> there is no reactor running, must be called from the context of a Tokio 1.x runtime
> note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
> Error logged during startup:
> there is no reactor running, must be called from the context of a Tokio 1.x runtime

### System Information
Paste the output of "mitmproxy --version" here.

`❯ mitmproxy --version`

> Mitmproxy: 10.4.0 binary
> Python:    3.12.4
> OpenSSL:   OpenSSL 3.2.2 4 Jun 2024
> Platform:  macOS-14.5-arm64-arm-64bit

**Comment by mhils** [comment]
I can't repro this over here.

1. How did you install mitmproxy?
2. What's your output for `mitmdump --mode local -vvv`?

**Comment by asadboi123** [comment]
Same here.

1. homebrew
2. 
```
thread '<unnamed>' panicked at mitmproxy-rs/src/server/local_redirector.rs:118:30:
there is no reactor running, must be called from the context of a Tokio 1.x runtime
note: run with "RUST_BACKTRACE=1" environment variable to display a backtrace
[04:41:46.018] there is no reactor running, must be called from the context of a Tokio 1.x runtime
Error logged during startup, exiting...

**Comment by mhils** [comment]
Is this a regression for either of you (i.e. did it work in a previous version), or you don't know?

**Comment by mhils** [comment]
I managed to track this down, we'll have a bugfix release ASAP! :)

**Comment by mhils** [comment]
This is fixed in 10.4.1, which should automatically hit Homebrew in the next day or so: https://github.com/Homebrew/homebrew-cask/pulls?q=is%3Apr+mitmproxy
